content/sdk/10.js/00.ethers/20.v6/03.providers.md

```diff
@@ -1430,10 +1323,9 @@ Only works for tokens bridged on default ZKsync Era bridges.
 
 #### Inputs
 
-| Parameter        | Type      | Description                                       |
-|------------------| --------- |---------------------------------------------------|
-| `token`          | `Address` | The address of the token on L1.                   |
-| `bridgeAddress?` | `Address` | The address of the custom bridge on L2(optional). |
+| Parameter | Type      | Description                     |
+| --------- | --------- | ------------------------------- |
+| `token`   | `Address` | The address of the token on L1. |
```

Comment:
Why is the `bridgeAddress` removed?
https://github.com/zksync-sdk/zksync-ethers/blob/main/src/provider.ts#L233